src/tests/ftest/pool/list_verbose.py

```diff
@@ -177,7 +177,7 @@ def verify_pool_lists(self, targets_disabled, scm_size, nvme_size):
             # Verify scm_size using the threshold rather than the exact match.
             rank_count = len(pool.target_list.value)
             if scm_size[index] is None:
-                created = pool.scm_size.value * rank_count
+                created = pool.scm_per_rank * rank_count
```

Comment:
This seems to ultimately compare scm size from `pool create` and `pool list`. Is that the expected behavior of this test?

Reply:
Yes, from what I understand. The issue there was to compare the size given to the command line and not the real SCM size of the created pool.
For the history, initially the dmg command was not returning this last size, but the size given to the command line. Thus, previously to the landing of the pull request [8864](https://github.com/daos-stack/daos/pull/8864), using pool.scm_size.value or pool.scm_per_rank was the same.
Please, @shimizukko could you confirm or rectify my statements.

Reply:
Fine with me. Thanks for the explanation!